server/src/main/kotlin/suwayomi/tachidesk/graphql/mutations/MangaMutation.kt

```diff
@@ -44,46 +45,60 @@ class MangaMutation {
         val patch: UpdateMangaPatch
     )
 
-    private fun updateMangas(ids: List<Int>, patch: UpdateMangaPatch) {
+    private suspend fun updateMangas(ids: List<Int>, patch: UpdateMangaPatch) {
```

Comment:
Can you revert most of the changes here? I had it like this so that we could add more user fields with minimal changes, just do another check for inLibrary when choosing what to do with the thumbnail

Reply:
done